app/src/main/java/eu/kanade/tachiyomi/ui/browse/source/browse/BrowseSourceController.kt

```diff
@@ -133,14 +133,15 @@ open class BrowseSourceController(bundle: Bundle) :
     override fun getTitle(): String? {
         return when (mode) {
             Mode.CATALOGUE -> presenter.source.name
-            Mode.RECOMMENDS -> recommendsConfig!!.origTitle
+            Mode.RECOMMENDS -> recommendsConfig!!.manga.title
         }
     }
 
     override fun createPresenter(): BrowseSourcePresenter {
         return BrowseSourcePresenter(
             args.getLong(SOURCE_ID_KEY),
-            if (mode == Mode.RECOMMENDS) recommendsConfig!!.origTitle else args.getString(SEARCH_QUERY_KEY),
+            args.getString(SEARCH_QUERY_KEY),
+            recommendsConfig?.manga,
```

Comment:
Wrap this in a if statement

Reply:
How exactly? I added a [parameter](https://github.com/jobobby04/TachiyomiSY/blob/045fc2b4e5c7c19da4378b340844499e31338191/app/src/main/java/eu/kanade/tachiyomi/ui/browse/source/browse/BrowseSourcePresenter.kt#L56) (searchManga) because the types of `searchQuery` and `searchManga` collideed.

Reply:
You can use sokething like searchManga = if(recommend is true) manga else null